dockstore-client/src/main/java/io/dockstore/client/cli/nested/NotificationsClients/Message.java

```diff
@@ -0,0 +1,31 @@
+package io.dockstore.client.cli.nested.NotificationsClients;
+
+/**
+ * Message structure based on what Slack accepts with the exception of the uuid
+ *
+ * @author gluu
+ * @since 19/01/18
+ */
+public class Message {
+    String text;
+
+    // Slack specific property to let the user know that the message came from Dockstore CLI
+    String username = "Dockstore CLI";
```

Comment:
username still looks arbitrary

Reply:
It's fixed as "Dockstore CLI" to specifically let the user know it came from the Dockstore CLI

Reply:
Ok, break this up. 
Add a property to the message called 'platform', call that `Dockstore CLI` and then make the username actually a username.